libs/core/src/lib/fixed-card-layout/fixed-card-layout.component.ts

```diff
@@ -100,11 +198,13 @@ export class FixedCardLayoutComponent implements OnInit, AfterContentInit, After
     cardDraggedDropped: EventEmitter<CardDropped> = new EventEmitter<CardDropped>();
 
     /** Array of CardDefinitionDirective Array.To make Table kind of layout.*/
-    public columns: CardColumn[];
+    columns: CardColumn[];
```

Comment:
Is it revealed to the user? Otherwise it should be `@hidden` and prefixed.

Reply:
done